README.md

```diff
@@ -86,6 +86,7 @@ Here is the default setup function. All of these values are optional, and if you
 require("gitlab").setup({
   port = nil, -- The port of the Go server, which runs in the background, if omitted or `nil` the port will be chosen automatically
   log_path = vim.fn.stdpath("cache") .. "/gitlab.nvim.log", -- Log path for the Go server
+  debug_type = "", -- "" (nothing), "request", "response", or "both" for the Go server logs
```

Comment:
I would maybe go for something like
```lua
debug = { go_request = false,  go_response = false }
```
so that it is  more extensible. The reason for that is that maybe later it would be also nice to have some debug / trace level of logging in lua.

Reply:
Made that change.